openshift/ci-operator/knative-images/kn/Dockerfile

```diff
@@ -10,8 +10,8 @@ COPY . .
 ENV CGO_ENABLED=1
 ENV GOEXPERIMENT=strictfipsruntime
 
-ENV KN_PLUGIN_FUNC_UTIL_IMAGE=registry.redhat.io/openshift-serverless-1/kn-plugin-func-func-util-rhel8@sha256:9881eac38d6186a5df36647a6d827d72c863e13e8dd9f48db94fcd4074237e9a
-ENV KN_PLUGIN_EVENT_SENDER_IMAGE=registry.redhat.io/openshift-serverless-1/kn-plugin-event-sender-rhel8@sha256:b25583a3082407a4734a5afe4376ff4f1de8baacbf771e4885d5cfe10d990de8
+ENV KN_PLUGIN_FUNC_UTIL_IMAGE=registry.redhat.io/openshift-serverless-1/kn-plugin-func-func-util-rhel8@sha256:c5c6cab0fcd79d2657fc70d87d8856bf16af08cc189a1c90a161360f71f046f0
+ENV KN_PLUGIN_EVENT_SENDER_IMAGE=registry.redhat.io/openshift-serverless-1/kn-plugin-event-sender-rhel8@sha256:89752446cce6a89fc830e081bed6da50f8ad5ac2920fbe7a16c74a87fbf8f00e
```

Comment:
Do you know why it picks up sha's wrong?  Not sure from where it fetches
/hold

Reply:
@dsimansk

Reply:
Fixed in https://github.com/openshift-knative/client/pull/811/commits/21bab278ebaa0cc1df1c8eec73365153c037ae59